docs/plugins.rst

```diff
@@ -202,6 +212,13 @@ definitions in Airflow.
     def stat_name_dummy_handler(stat_name):
         return stat_name
 
+    # A global operator extra link that redirect you to google
+    class GoogleLink(BaseOperatorLink):
```

Comment:
same. It is not intuitive for user why using this feature with the google link in the doc. Maybe using the kibana log example or qubole url example?

Reply:
This is for the testing purpose so I want to make it as simple as possible